__tests__/basic.js

```diff
@@ -20,3 +20,16 @@ test('setup', async () => {
 
   expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
 });
+
+test('long urls', async () => {
```

Comment:
```suggestion
test('long URLs', async () => {
```

I believe the snapshot would work (Jest should lowercase the name).